Modules/IO/ImageBase/include/itkImageSeriesReader.h

```diff
@@ -209,6 +213,8 @@ class ITK_TEMPLATE_EXPORT ImageSeriesReader : public ImageSource<TOutputImage>
 
   bool m_SpacingDefined{ false };
 
+  double m_SpacingWarningRelThreshold{ 1e-6 };
```

Comment:
Nitpick: a hard coded number is probably fine for most use cases, but @lassoan 's original suggestion of 1% of slice spacing (or 0.1% even if you must) would be much more practical.  For both CT and MR scans there are physical reasons why the table position or slice reconstruction intervals may have discrete but negligible errors that aren't due simply to floating point rounding.

Reply:
I agree with the observation indicated by @pieper.  Personal preference is to err on the side of 0.1% until we identify that this is too stringent, and then increase based on data.